object/object.go

```diff
@@ -58,25 +58,19 @@ type Object struct {
 	pld    []byte
 }
 
-// RequiredFields contains the minimum set of object data that must be set
-// by the NeoFS user at the stage of creation.
-type RequiredFields struct {
-	// Identifier of the NeoFS container associated with the object.
-	Container cid.ID
+// New creates and initializes new [Object] with the current version from SDK
+// and given mandatory fields.
+func New(cnr cid.ID, owner user.ID) *Object {
```

Comment:
any version i have seen so far doesn't seem sufficient to me, but i have nothing to suggest either, so let it set the version at least now